cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/DRep.hs

```diff
@@ -125,6 +124,12 @@ toLedgerDelegatee t =
       conwayEraOnwardsConstraints cOn $
         right $ Ledger.DelegVote $ Ledger.DRepCredential $ Ledger.ScriptHashObj scriptHash
 
+toDelegateeEnvelope :: Ledger.Delegatee ledgerera -> TextEnvelopeDescr
+toDelegateeEnvelope = \case
+  Ledger.DelegStake{} -> "Stake Delegation Certificate"
+  Ledger.DelegVote{} -> "Vote Delegation Certificate"
+  Ledger.DelegStakeVote{} -> "Stake and Vote Delegation Certificate"
```

Comment:
Should add a mention that the description renamed from `Stake Address Delegation Certificate` to `Stake Delegation Certificate` to the change logs.

Reply:
done